tests/test_charms.py

```diff
@@ -106,7 +106,7 @@ async def test_build_and_deploy(ops_test, lightkube_client, dummy_resources_for_
 
     # Deploy jupyter-controller, admission-webhook, kubeflow-profiles and kubeflow-dashboard
     await ops_test.model.deploy(controller_charm, resources={"oci-image": controller_image_path})
-    await ops_test.model.deploy("admission-webhook", channel="latest/edge")
+    await ops_test.model.deploy("admission-webhook", channel="latest/edge", trust=True)
     await ops_test.model.deploy("kubeflow-profiles", channel="latest/edge", trust=True)
     await ops_test.model.deploy("kubeflow-dashboard", channel="latest/edge", trust=True)
     await ops_test.model.add_relation("kubeflow-profiles", "kubeflow-dashboard")
```

Comment:
maybe out of scope for this pr - should we pin these to a tracks from this release rather than `latest`?  admission-webhook breaks us here because we did not pin our CI when cutting the track before.

Reply:
I thought about it, but I want to do minimal amount of changes.